generate-certs

```diff
@@ -66,7 +66,7 @@ cp ${BASE_SSL_CONF} ${conf}
 
 openssl genrsa -out private/koji_ca_cert.key 2048
 
-sed -i "s/email\:move/DNS.1:koji-hub,DNS.2:${KOJI_HUB_IP}/" ${conf}
+sed -i "s/email\:move/DNS.1:koji-hub,IP.1:${KOJI_HUB_IP}/" ${conf}
```

Comment:
I remember having to mark it as `DNS` before. My lack of explaining comment this doesn't help clarify this counter intuitive usage of `DNS` :(

What is the issue that is being addressed by this?

Reply:
The issue this addresses comes down to the fact that web browsers and clients are not supposed to be able to verify IP addresses with a DNS.N entry.

Reply:
I get this when I try this out:
```
SSLError: hostname '172.17.0.1' doesn't match 'koji-hub'
```